Formula/sbt.rb

```diff
@@ -20,9 +20,13 @@ def install
       s.gsub! "/etc/sbt/sbtopts", "#{etc}/sbtopts"
     end
 
-    inreplace "bin/sbt-launch-lib.bash", "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+    inreplace "bin/sbt-launch-lib.bash" do |s|
+      s.gsub! "$(dirname \"$(realpath \"$0\")\")", "#{libexec}/bin"
```

Comment:
Is there an upstream issue tracking "abstracting" this? Note that `realpath` won't work on macOS regardless of Homebrew since it's not even a command in the BSD userland, only a C function

```
     #include <stdlib.h>

     char *
     realpath(const char *restrict file_name, char *restrict resolved_name);
```